src/local_device.rs

```diff
@@ -1,7 +1,10 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
+use std::path::PathBuf;
+
+use libparsec::client_types;
```

Comment:
Same:
- others crate
- libparsec
- crate/super

Reply:
https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#group_imports

Reply:
I did run `rustfmt` through `pre-commit` :thinking:

Reply:
Looks like the default behavior of `rustfmt` is to preserve order and just sort blocks themselves. I guess we can configure it to enforce the order we want everywhere.